charts/waterfallchart/src/styling-panel-property-definiton.js

```diff
@@ -6,19 +6,16 @@
  */
 const getStylingItems = (flags, styleOptions) => {
   const items = {};
```

Comment:
There is no need to define an empty object first here, you can just populate it with the axis properties right away

Reply:
Thanks, fixed it now.